### PR TITLE
DB-8197: validate backup

### PR DESCRIPTION
**Pull request by jyuanca**

### CODE REVIEW

**Comment by jyuanca** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Comment by jyuanca** [comment]
Please also review code changes for ee branch

File: hbase_sql/hbase1.2.0/src/main/java/com/splicemachine/hbase/BackupEndpointObserver.java [hunk 1]
```diff
@@ -154,7 +156,10 @@ public void prepareBackup(
                 canceled = BackupUtils.backupCanceled();
                 if (!canceled) {
                     // Create a ZNode to indicate that the region is being copied
-                    boolean created = ZkUtils.recursiveSafeCreate(regionBackupPath, HConfiguration.BACKUP_IN_PROGRESS, ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
+                    HRegionInfo regionInfo = region.getRegionInfo();
```
Comment (dgomezferro) [inline hbase_sql/hbase1.2.0/src/main/java/com/splicemachine/hbase/BackupEndpointObserver.java]: We need to apply this change to other platforms too, right?

Comment (jyuanca) [reply hbase_sql/hbase1.2.0/src/main/java/com/splicemachine/hbase/BackupEndpointObserver.java]: @dgomezferro good catch!

**Pull request merged**